### ISSUE TITLE
PHP Strict standards:  Declaration of Acquia\Blt\Robo\Config\YamlConfigProcessor

### ISSUE THREAD

**Issue by justclint** [acquia/blt#1753]
My system information:
* Operating system type: OSX
* Operating system version: 10.12.5
* BLT version: 8.8.6

Output of `blt doctor`:
```
| Property             | Value                                                                            |
+----------------------+----------------------------------------------------------------------------------+
| %paths               |                                                                                  |
|  - %root             | /Users/clinta/www/sites/padi-blt/docroot                                         |
|  - %site             | sites/default                                                                    |
|  - %modules          | modules                                                                          |
|  - %themes           | themes                                                                           |
|  - %config-sync      | /Users/clinta/www/sites/padi-blt/config/default                                  |
| blt-version          | 8.8.6                                                                            |
| config-sync          | /Users/clinta/www/sites/padi-blt/config/default                                  |
| drupal-settings-file | sites/default/settings.php                                                       |
| drupal-version       | 8.3.4                                                                            |
| drush-alias-files    | /Users/clinta/.drush/aliases.drushrc.php                                         |
|                      | /Users/clinta/.drush/padi.aliases.drushrc.php                                    |
|                      | /Users/clinta/.drush/padiblt.aliases.drushrc.php                                 |
|                      | /Users/clinta/.drush/padisandbox.aliases.drushrc.php                             |
|                      | /Users/clinta/www/sites/padi-blt/docroot/../drush/site-aliases/aliases.drushrc.p |
|                      | hp                                                                               |
|                      | /Users/clinta/www/sites/padi-blt/docroot/../drush/site-aliases/example.acsf.alia |
|                      | ses.drushrc.php                                                                  |
|                      | /Users/clinta/www/sites/padi-blt/docroot/../drush/site-aliases/example.local.ali |
|                      | ases.drushrc.php                                                                 |
| drush-conf           | /Users/clinta/www/sites/padi-blt/drush/drushrc.php                               |
| drush-script         | /Users/clinta/www/sites/padi-blt/vendor/drush/drush/drush.php                    |
| drush-temp           | /tmp                                                                             |
| drush-version        | 8.1.12                                                                           |
| modules              | modules                                                                          |
| php-bin              | /usr/local/bin/php                                                               |
| php-conf             | /usr/local/etc/php/5.6/php.ini                                                   |
| php-os               | Darwin                                                                           |
| root                 | /Users/clinta/www/sites/padi-blt/docroot                                         |
| site                 | sites/default                                                                    |
| themes               | themes                                                                           |
| uri                  | http://default                                                                   |
+----------------------+----------------------------------------------------------------------------------+
+--------------------------+----------------------------------------------------------------------------------+
| Check                    | Outcome                                                                          |
+--------------------------+----------------------------------------------------------------------------------+
| checkDocrootExists       | Found docroot.                                                                   |
| checkCoreExists          | Drupal core exists                                                               |
| checkSettingsFile        | BLT settings are included in settings file.                                      |
| checkLocalSettingsFile   | Could not find local settings file.                                              |
|                          | Your local settings file should exist at                                         |
|                          | /Users/clinta/www/sites/padi-blt/docroot/sites/default/settings/local.settings.p |
|                          | hp.                                                                              |
| checkLocalDrushFile      | Local drushrc file does not exist.                                               |
|                          | Create /Users/clinta/www/sites/padi-blt/docroot/sites/default/local.drushrc.php. |
|                          | Run `blt setup:drush:settings` to generate it automatically, or run `blt setup`  |
|                          | to run the entire setup process.                                                 |
| checkFileSystem:%files   | Public files directory is not set.                                               |
| checkFileSystem:%private | Private files directory is not set.                                              |
| checkFileSystem:%temp    | Temporary files directory is not set.                                            |
| checkDbConnection        | Could not connect to MySQL database.                                             |
|                          |                                                                                  |
|                          | Is your *AMP stack running?                                                      |
|                          | Are your database credentials correct?                                           |
|                          |   db-driver:                                                                     |
|                          |   db-hostname:                                                                   |
|                          |   db-username:                                                                   |
|                          |   db-password:                                                                   |
|                          |   db-name:                                                                       |
|                          |   db-port:                                                                       |
|                          |                                                                                  |
|                          | Are you using the correct PHP binary?                                            |
|                          | Is PHP using the correct MySQL socket?                                           |
|                          |   php-os: Darwin                                                                 |
|                          |   php-bin: /usr/local/bin/php                                                    |
|                          |   php-conf: Array                                                                |
|                          |   php-mysql:                                                                     |
|                          |                                                                                  |
|                          | Are you using the correct site and settings.php file?                            |
|                          |   site: sites/default                                                            |
|                          |   drupal-settings-file: sites/default/settings.php                               |
|                          |                                                                                  |
|                          | To verify, run `drush sqlc`                                                      |
|                          |                                                                                  |
| checkDrupalBootstrapped  | Could not bootstrap Drupal via drush without alias.                              |
| checkDrupalInstalled     | Drupal is not installed.                                                         |
|                          |                                                                                  |
|                          | Run `blt local:setup` to install Drupal locally.                                 |
| checkCaching:page        | Drupal cache is disabled.                                                        |
| checkCaching:css         | CSS preprocessing is disabled.                                                   |
| checkCaching:js          | JS preprocessing is disabled.                                                    |
| checkNvmExists           | NVM exists.                                                                      |
| checkComposer:require    | acquia/blt is in composer.json's require object.                                 |
| checkComposer:plugins    | hirak/prestissimo plugin for composer is not installed.                          |
|                          |   Run `composer global require hirak/prestissimo:^0.3` to install it.            |
|                          |   This will improve composer install/update performance by parallelizing the     |
|                          | download of dependency information.                                              |
| checkBehat:exists        | tests/behat/local.yml is missing!                                                |
|                          |   Run `blt setup:behat` to generate it from example.local.yml.                   |
| checkProjectYml:keys     | project.yml has no deprecated keys.                                              |
| checkPhpDateTimezone     | PHP setting for date.timezone is not set.                                        |
|                          |   Define date.timezone in /usr/local/etc/php/5.6/php.ini   
```

When I run this command:
```
blt vm
```

I get the following output:
```
PHP Strict standards:  Declaration of Acquia\Blt\Robo\Config\YamlConfigProcessor::preprocess() should be compatible with Consolidation\Config\Loader\ConfigProcessor::preprocess(array $config) in /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/src/Robo/Config/YamlConfigProcessor.php on line 9
PHP Stack trace:
PHP   1. {main}() /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:0
PHP   2. require_once() /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:25
PHP   3. spl_autoload_call() /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:45
PHP   4. Composer\Autoload\ClassLoader->loadClass() /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:45
PHP   5. Composer\Autoload\includeFile() /Users/clinta/www/sites/padi-blt/vendor/composer/ClassLoader.php:322
PHP   6. include() /Users/clinta/www/sites/padi-blt/vendor/composer/ClassLoader.php:444

Strict standards: Declaration of Acquia\Blt\Robo\Config\YamlConfigProcessor::preprocess() should be compatible with Consolidation\Config\Loader\ConfigProcessor::preprocess(array $config) in /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/src/Robo/Config/YamlConfigProcessor.php on line 9

Call Stack:
    0.0003     244432   1. {main}() /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:0
    0.1048    1766416   2. require_once('/Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt-robo-run.php') /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:25
    0.1776    2454520   3. spl_autoload_call() /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:45
    0.1776    2454584   4. Composer\Autoload\ClassLoader->loadClass() /Users/clinta/www/sites/padi-blt/vendor/acquia/blt/bin/blt:45
    0.1776    2454752   5. Composer\Autoload\includeFile() /Users/clinta/www/sites/padi-blt/vendor/composer/ClassLoader.php:322
    0.1802    2458848   6. include('/Users/clinta/www/sites/padi-blt/vendor/acquia/blt/src/Robo/Config/YamlConfigProcessor.php') /Users/clinta/www/sites/padi-blt/vendor/composer/ClassLoader.php:444
```

And I expected this to happen:
I just had to do a fresh install of our repo and ran into this error. Its been a while since Ive had to run 'blt vm' so this is the first time running into this. 

I saw this issue https://github.com/acquia/blt/pull/1747 which is similar but not exactly the same but maybe its related.

If theres any additional info I could provide to help, just let me know.

**Comment by justclint** [comment]
UPDATE: I continued with the build process and after some testing, the VM does seem to get built as expected so that error doesn't seem to have any negative affect on the VM. 

But I do continue to get that error every time I try a fresh installation.

**Comment by grasmash** [comment]
@justclint Can you try just running `composer update`? I think you may just need a new version of Robo.

**Comment by grasmash** [comment]
Resolved by #1766
